src/Interfaces/ImageInterface.php

```diff
@@ -66,7 +66,7 @@ public function size(): SizeInterface;
      * @param EncoderInterface $encoder
      * @return EncodedImageInterface
      */
-    public function encode(EncoderInterface $encoder): EncodedImageInterface;
+    public function encode(EncoderInterface $encoder = new AutoEncoder()): EncodedImageInterface;
```

Comment:
This refers to `Intervention\Image\Interfaces\AutoEncoder`, which does not exist.

Reply:
Sorry about that, monday.. Fixed in the last commit.